pyramid_swagger/exceptions.py

```diff
@@ -8,3 +11,12 @@ class RequestValidationError(HTTPClientError):
 
 class ResponseValidationError(HTTPInternalServerError):
     pass
+
+
+def wrap_exception(method):
```

Comment:
One thought: let's name this specifically or pass in the exception to reraise as.